packages/css-reset/README.md

```diff
@@ -1,3 +1,7 @@
-# TDS Core: CSS Reset
+```css noeditor
+@import '~@tds/core-css-reset/index';`
+```
+
+Version: 0.0.0-development
 
 Global CSS rules to apply consistent base styles, including a CSS reset, font faces, and root styles.
```

Comment:
Let's include more info here. Let's include the stuff from the FAQ so it's all in one place.

Reply:
done :)